flycheck.el

```diff
@@ -9,7 +9,7 @@
 ;; URL: http://www.flycheck.org
 ;; Keywords: convenience, languages, tools
 ;; Version: 31-cvs
-;; Package-Requires: ((dash "2.12.1") (pkg-info "0.4") (let-alist "1.0.4") (seq "1.11") (emacs "24.3"))
+;; Package-Requires: ((dash "2.12.1") (pkg-info "0.4") (let-alist "1.0.4") (seq "1.11") (emacs "24.4"))
```

Comment:
Sorry, my bad. Got warning that several functions require 24.4. Just today I realized that those are also implemented here for users on 24.3.